test/services/helpers/service.test.js

```diff
@@ -4,7 +4,7 @@ const nock = require('nock');
 const app = require('../../../src/app');
 
 // eslint-disable-next-line import/no-dynamic-require
-const config = require(`../../../config/${process.env.NODE_ENV || 'default'}.json`);
+const config = require(`../../../config/${process.env.NODE_ENV || 'default'}.json`); // TODO cleanup
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected use of process.env.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5246313)